### DISCUSSION TITLE
Requesting author list for citation

### DISCUSSION THREAD

**Discussion by drewoldag** [copier-org/copier#1542] [Q&A] [answered]
Hello copier maintainers! Our team has built a copier template and we're planning to submit a paper about it. We would like to give appropriate credit to copier and its maintainers. We found the list of authors on the Credits page in the documentation, and just wanted to double check that the author order we extracted from that is correct.

Here's what we have:
```
@misc{copier,
  author = {Juan-Pablo Scaletti and Ben Felder and Jairo Llopis and Timothée Mazzucotelli and Sigurd Spieckermann and Copier Contributors},
  title = {Copier},
  year = {2024},
  publisher = {GitHub},
  journal = {GitHub repository},
  url = {https://github.com/copier-org/copier}
}
```

If there is a preferred author list or if this one is incomplete/out of order, please provide the preferred list so that we can include it, thanks!

**Comment by pawamoy** [comment, marked as answer]
That looks right! Can you share a link to the paper once published :wink:?

**Comment by drewoldag** [comment]
Awesome, thanks for the confirmation! I'll be happy to send a link to the paper when it's published!

**Comment by drewoldag** [comment]
Hi all, it took quite a while to get the paper published, but it finally made it through the process. If you're interested in taking a look it's available here: https://iopscience.iop.org/article/10.3847/2515-5172/ad4da1
